src/features/transactions/wrapEventSubscriber.ts

```diff
@@ -0,0 +1,54 @@
+import { Web3Provider } from "@ethersproject/providers";
+import { Dispatch } from "@reduxjs/toolkit";
+
+import { Contract } from "ethers";
+
+import { store } from "../../app/store";
+import {
+  decrementBalanceBy,
+  incrementBalanceBy,
+} from "../balances/balancesSlice";
+import { SubmittedTransaction } from "./transactionsSlice";
+
+const handleWrapEvent = (eventName: string, data: any, dispatch: any) => {
```

Comment:
`const handleWrapEvent = (data: any, dispatch: Dispatch<Action>) => {`

eventName is not used, so it can be omitted.

Reply:
Okay! I used that for logging, but no point in keeping it around